src/main/res/values/strings.xml

```diff
@@ -456,45 +458,45 @@
     <string name="menu_add_relation_member">Add member</string>
     <string name="menu_select_relation_members">Select members</string>
     <string name="menu_tag">Edit tags</string>
-    <string name="menu_openstreetbug">Add Note</string>
+    <string name="menu_openstreetbug">Add note</string>
     <string name="menu_easyedit">EasyEdit</string>
     <string name="menu_camera">Camera</string>
-    <string name="menu_gps">GPS…</string>
+    <string name="menu_gps">Positioning…</string>
     <string name="menu_gps_show">Show location</string>
     <string name="menu_gps_follow">Follow location</string>
-    <string name="menu_gps_goto">Goto current location</string>
-    <string name="menu_gps_start">Start GPS track</string>
-    <string name="menu_gps_pause">Pause GPS track</string>
-    <string name="menu_gps_clear">Clear GPS track and waypoints</string>
-    <string name="menu_gps_track_managment">Track management...</string>
+    <string name="menu_gps_goto">Go to current location</string>
+    <string name="menu_gps_start">Start positional track</string>
```

Comment:
Should this and the following two strings not be GPX* instead of GPS?

Reply:
Yes that should be "GPX".